checkov/terraform/checks/resource/kubernetes/CPULimits.py

```diff
@@ -21,6 +21,8 @@ def scan_resource_conf(self, conf: dict[str, list[Any]]) -> CheckResult:
         spec = conf['spec'][0]
 
         containers = spec.get("container")
+        if not containers:
+            return CheckResult.PASSED
```

Comment:
```suggestion
            return CheckResult.UNKNOWN
```
fits better, because it actually doesn't apply to the resource without having containers